packages/avatar/src/vars/index.js

```diff
@@ -7,15 +16,13 @@ export const sizes = {
 }
 
 const colors = [
-  '#49897C',
-  '#9A171B',
-  '#1F964E',
-  '#076F84',
-  '#BE344B',
-  '#2D4E82',
-  '#6D8C36',
-  '#AE8018',
-  '#82214E'
+  colorsBlue[7],
+  colorsTeal[7],
+  colorsGreen[7],
+  colorsOrange[7],
+  colorsRed[7],
+  colorsPink[7],
+  colorsPurple[7]
```

Comment:
nitpick. colorsBlue['base'] is more readable for me. but whatever 😉

Reply:
Base is index 6 not 7 Brian has it at 7in slack convo